go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/hako/durafmt v0.0.0-20210608085754-5c1018a4e16b
 	github.com/jbowtie/gokogiri v0.0.0-20190301021639-37f655d3078f
 	github.com/jbowtie/ratago v0.0.0-20200401224626-3140c0a9b186
+	github.com/jedib0t/go-pretty/v6 v6.5.9
```

Comment:
I think that it should be assessed whether we want to add a new external dependency. How much it increases binary sizes? Do we really need it? And if so, whether we can make use of it in other places.

Reply:
removed the dependency and used Println instead